code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -517,10 +513,12 @@
 		var/mob/living/simple_animal/hostile/H = M
 		H.AIStatus = initial(H.AIStatus)
 
-
 /obj/effect/timestop/wizard
 	duration = 100
 
+/obj/effect/timestop/wizard/New()
+	..()
+	timestop()
```

Comment:
This would break cult timestop. Instead of removing timestop() from all New(), why not just make the sepia timestop a subtype and then avoid calling timestop() on new() for that one instead?